Lesson05/banking.js

```diff
@@ -0,0 +1,90 @@
+
+/*The user should see a prompt so they can type which action to perform.
+The user will have a list of 4 actions to choose from.
+Enter Q to quit (immediately quits the program).
+Enter W to withdraw.
+The user will be prompted again to enter an amount to withdraw. After withdrawing money, they should be able to type another option.
+Enter D to deposit.
+The user will be prompted again to enter an amount to deposit. After depositing money, they should be able to type another option.
+Enter B to view balance.
+The user will see their balance. Afterwards, they should be able to type another option.
+The program will loop asking for input until the user enters Q.
+*/
+
+function promptEnterCode() {
+    let answer = prompt('Enter one of the following codes: Q, W, B, or D');
+    return(answer);
+}
+
+function handleAction() {
+    let accountBalance = 1000;
+    let input = promptEnterCode();
+
+    for (let i = 0; i >= 0; i++) {
```

Comment:
this is pretty fragile. It might work, but could go wrong if you're not careful. A while loop would be better, and you flip your variable when you quit